src/lib/components/table/components/TableCell/BaseCell/BaseCell.tsx

```diff
@@ -80,7 +80,8 @@ export const BaseCell = ({
         className={classNames(
           'd-flex fd-column',
           alignClassName,
-          styles.relative
+          styles.relative,
+          { [styles.maxWidth]: align === 'center' }
```

Comment:
Do you think this should only be applied when there is modalContent?

Reply:
What you mean is not applying this style if the content is a Checkmark or Rating?

Reply:
Yes, I thin Checkmark or Rating should be used with a description that's shown below, not on the side, what do you think?

Reply:
Ok I get what you mean. I'll make the change.